tests/deepsparse/transformers/pipelines/test_text_generation.py

```diff
@@ -449,6 +451,51 @@ def test_streaming_mode_returns_generator(self, setup):
         ), "Pipeline should return a generator of output_schema \
                objects in streaming mode"
 
+    @helper_test
+    def test_inference_no_kv_cache_deepsparse(self, setup):
+        self._test_inference_no_kv_cache()
+
+    @helper_test
+    def test_inference_no_kv_cache_ort(self, setup):
+        self._test_inference_no_kv_cache(engine_type="onnxruntime")
+
+    def _test_inference_no_kv_cache(self, **kwargs):
+        engine_type = kwargs.get("engine_type", "deepsparse")
+        model_path_no_cache = self._get_model_path_no_cache()
+        pipeline = self.get_pipeline(
+            model_path=model_path_no_cache, engine_type=engine_type
+        )
+        assert not pipeline.cache_support_enabled, (
+            "This pipeline test inference using non-kv cache "
+            "model and thus should not support kv cache"
+        )
+        output = self.run_pipeline(pipeline, max_length=1)
+        # prompt logits + one logit for the new generated token
+        logits = output.generations[0].score[-(self.prompt_length + 1) :, :]
+        generated_logits, prompt_logits, *_ = self.torch_ground_truth
+        logits_gt = numpy.concatenate(
+            [prompt_logits[0], generated_logits[0, :1, :]], axis=0
+        )
+        assert numpy.allclose(logits, logits_gt, atol=self.precision)
+
+    def _get_model_path_no_cache(self):
+        from sparsezoo import Model
+
+        if not self.model_path.startswith("zoo:"):
+            pytest.skip("For this test, for now only the zoo model is supported")
+        training_dir_path = Model(self.model_path).training.path
```

Comment:
we don't want to download the entire training directory just to get one file, especially since the files can be so large, let's use `.get_file` instead? if already downloaded for torch files, that's still ok, just don't see a reason to force it here

Reply:
good point. I will only download those files that are absolutely necessary.